checker/build.xml

```diff
@@ -297,6 +288,11 @@
             <package name="org.checkerframework.common.*"/>
             <package name="org.checkerframework.checker.*"/>
 
+            <fileset dir="${checker.loc}/${src}">
+                <include name="**/*.java"/>
+                <exclude name="${checker.jdk8orhigher.sources}" if="${jdk7orlower}"/>
```

Comment:
How can we make sure that Opt appears in the Javadoc?  It would be better to always build the Javadoc using Java 8 if it is available.  That seems fine even if javac uses Java 7.  Furthermore, when making a release it should be required that Java 8 is available.